src/data/index.ts

```diff
@@ -141,291 +141,339 @@ const json: RequirementsJson = {
       schools: ['AG', 'AS1', 'AS2'],
       requirements: bioRequirements,
       advisors: bioAdvisors,
+      abbrev: 'Bio',
     },
     BSOC: {
       name: 'Biology and Society',
       schools: ['AG', 'AS1', 'AS2'],
       requirements: bsocRequirements,
       advisors: bioAdvisors,
+      abbrev: 'BSoc',
     },
     BME: {
       name: 'Biomedical Engineering',
       schools: ['EN'],
       requirements: biomedicalEngineeringRequirements,
       advisors: biomedicalEngineeringAdvisors,
+      abbrev: 'BME',
     },
     BE: {
       name: 'Biological Engineering',
       schools: ['EN'],
       requirements: bioEngineeringRequirements,
       advisors: bioEngineeringAdvisors,
+      abbrev: 'BioEng',
     },
     CE: {
       name: 'Civil Engineering',
       schools: ['EN'],
       requirements: civilRequirements,
       advisors: civilAdvisors,
+      abbrev: 'Civil Eng',
     },
     CHEM: {
       name: 'Chemistry',
       schools: ['AS1', 'AS2'],
       requirements: chemRequirements,
       advisors: chemAdvisors,
+      abbrev: 'Chem',
     },
     CHEME: {
       name: 'Chemical and Biomolecular Engineering',
       schools: ['EN'],
       requirements: chemERequirements,
       advisors: chemEAdvisors,
+      abbrev: 'ChemE',
     },
     COMM: {
       name: 'Communication',
       schools: ['AG'],
       requirements: commRequirements,
       advisors: commAdvisors,
+      abbrev: 'Comm',
     },
     CRP: {
       name: 'City and Regional Planning',
       schools: ['AR'],
       requirements: crpRequirements,
       advisors: crpAdvisors,
+      abbrev: 'CRP',
     },
     CS: {
       name: 'Computer Science',
       schools: ['EN', 'AS1', 'AS2'],
       requirements: csRequirements,
       specializations: [MATH2940, CHEM2080],
       advisors: csAdvisors,
+      abbrev: 'CS',
     },
     DEA: {
       name: 'Design and Environmental Analysis',
       schools: ['HE'],
       requirements: deaRequirements,
       advisors: deaAdvisors,
+      abbrev: 'D+EA',
     },
     EAS: {
       name: 'Earth and Atmospheric Sciences',
       schools: ['AG', 'AS1', 'AS2', 'EN'],
       requirements: easRequirements,
       advisors: easAdvisors,
+      abbrev: 'EAS',
     },
     ECON: {
       name: 'Economics',
       schools: ['AS1', 'AS2'],
       requirements: economicsRequirements,
       advisors: economicsAdvisors,
+      abbrev: 'Econ',
     },
     ECE: {
       name: 'Electrical and Computer Engineering',
       schools: ['EN'],
       requirements: eceRequirements,
       advisors: eceAdvisors,
+      abbrev: 'ECE',
     },
     ENGL: {
       name: 'English',
       schools: ['AS1', 'AS2'],
       requirements: englishRequirements,
       advisors: englishAdvisors,
+      abbrev: 'English',
     },
     ENV: {
       name: 'Environmental Engineering',
       schools: ['EN'],
       requirements: envEngineeringRequirements,
       advisors: envEngineeringAdvisors,
+      abbrev: 'EnvEng',
```

Comment:
is there any reason some abbreviations are space separated while others are PascalCase?